primefaces/src/main/java/org/primefaces/component/dialog/Dialog.java

```diff
@@ -51,14 +51,14 @@ public class Dialog extends DialogBase {
 
     public static final String COMPONENT_TYPE = "org.primefaces.component.Dialog";
 
-    public static final String CONTAINER_CLASS = "ui-dialog ui-widget ui-widget-content ui-corner-all ui-shadow ui-hidden-container";
-    public static final String TITLE_BAR_CLASS = "ui-dialog-titlebar ui-widget-header ui-helper-clearfix ui-corner-top";
+    public static final String CONTAINER_CLASS = "ui-dialog ui-widget ui-widget-content  ui-shadow ui-hidden-container";
+    public static final String TITLE_BAR_CLASS = "ui-dialog-titlebar ui-widget-header ui-helper-clearfix ";
     public static final String TITLE_CLASS = "ui-dialog-title";
-    public static final String TITLE_BAR_CLOSE_CLASS = "ui-dialog-titlebar-icon ui-dialog-titlebar-close ui-corner-all";
+    public static final String TITLE_BAR_CLOSE_CLASS = "ui-dialog-titlebar-icon ui-dialog-titlebar-close ";
```

Comment:
those strings now all ends with a empty space

Reply:
whoops i was conservative with my search and replace for exact strings :)  Let me fix